apps/web/src/pages/_document.tsx

```diff
@@ -23,25 +23,6 @@ class LensterDocument extends Document {
             content="default"
           />
           <meta name="apple-mobile-web-app-title" content="Lenster" />
-
-          {/* Icons */}
```

Comment:
We added these because they are actually server rendered 🙇🏼

Reply:
Yes, and they are server-rendered when we use them in _app.tsx as well.
I couldn't keep it in _document because it caused tag duplication described [here](https://stackoverflow.com/questions/68204973/avoid-duplicate-meta-description-and-keywords-in-next-js#:~:text=Tags%20in%20_document%20will%20not%20be%20replaced%20(can%20be%20duplicated)%20even%20with%20a%20key%20identifier)